microprofile/metrics-smallrye/src/main/java/org/wildfly/extension/microprofile/metrics/_private/MicroProfileMetricsLogger.java

```diff
@@ -39,7 +39,7 @@
  *
  * @author <a href="http://jmesnil.net/">Jeff Mesnil</a> (c) 2018 Red Hat inc.
  */
-@MessageLogger(projectCode = "WFLYMETRICS", length = 4)
+@MessageLogger(projectCode = "WFLYMPMETRICS", length = 4)
```

Comment:
I changed the project code but we are fine as the new `metrics` subsystem reuse the `WFLYMETRICS` code and keep the same message id for the same errors and logs